libraries/wallet/wallet.cpp

```diff
@@ -2924,6 +2924,11 @@ account_history_operation_detail wallet_api::get_account_history_by_operations(s
     return result;
 }
 
+std::map<string,full_account> wallet_api::get_full_accounts( const vector<string>& names_or_ids, bool subscribe)
+{
+    return my->_remote_db->get_full_accounts(names_or_ids, false);
```

Comment:
here the subscribe bool will be always off.

Reply:
ah oh.. how careless I am, thanks for pointing out~ have fixed and pushed again